src/index.ts

```diff
@@ -40,11 +41,12 @@ export class Probot {
     this.logger = logger
     this.apps = []
     this.webhook = new Webhooks({ path: options.webhookPath, secret: options.secret })
-    if (options.githubToken) {
-      this.githubToken = options.githubToken
-      this.app = () => ''
-    } else {
-      this.app = createApp({ id: options.id!, cert: options.cert! })
+    this.githubToken = options.githubToken
```

Comment:
Do you no longer need to check if `options.githubToken` is defined? As done in line 43 of the removed block.

https://github.com/probot/probot/blob/43acbbc843f4df45e2243762d4f8fdc4e2ec81da/src/index.ts#L43

Reply:
The way `this.app()` works is different now, so we can simply always set `this.githubToken`. 

We later set `const token = this.githubToken || this.app.getSignedJsonWebToken()` https://github.com/probot/probot/pull/828/files#diff-8f42631f8d7d8cbaf47074d3ea5536d1R213

Reply:
There are also tests testing `githubToken` so I’m confident that this is still working

Reply:
Thanks for the clarification

Reply:
Thanks for your review :)